civitai-downloader-v2/src/cli/main.py

```
@@ -109,68 +110,648 @@ def cli(ctx, config, verbose):
+                allow_commercial = result.get("allowCommercialUse", [])
+                if not isinstance(allow_commercial, list):
+                    allow_commercial = []
+                
+                # Convert to binary format (1/0)
+                image_allowed = 1 if "Image" in allow_commercial else 0
+                rent_allowed = 1 if "Rent" in allow_commercial else 0
+                rent_civit_allowed = 1 if "RentCivit" in allow_commercial else 0
+                sell_allowed = 1 if "Sell" in allow_commercial else 0
+                
+                # Format numbers for display
+                downloads_display = f"{downloads//1000}k" if downloads >= 1000 else str(downloads)
```

Ruff (0.12.2)

Local variable `downloads_display` is assigned to but never used

Remove assignment to unused variable `downloads_display`

(F841)